app/controllers/api/v1/posters_controller.rb

```diff
@@ -1,7 +1,15 @@
 class Api::V1::PostersController < ApplicationController
     def index
-        posters = Poster.all
-        render json: PosterSerializer.new(posters) 
+        if params[:sort] 
+            posters = Poster.order(created_at: params[:sort].to_sym)
+        else
+            posters = Poster.all
+        end
+
+        metaObject = {
```

Comment:
defines meta hash that will be included in our serialized response, this aligns with JSON specifications.